### DISCUSSION TITLE
How to generate Zod schemas from smart contract function signatures

### DISCUSSION THREAD

**Discussion by 0xaaiden** [wagmi-dev/abitype#190] [Q&A]
I'm trying to generate Zod schemas from smart contract function signatures and generate dynamically a form on a webpage to fill in inputs for a function.

**Comment by Raiden1411** [comment]
Are you grabbing the ABI from a 3rd party api? Or do you have the human readable signatures? If you can provide an example of what you are trying to do that would be helpful.
